Library/Source/RuntimeImpl.cpp

```diff
@@ -34,7 +38,16 @@ namespace babylon
 
     void RuntimeImpl::Suspend()
     {
-        m_engine->Suspend();
```

Comment:
This seems like something that should be a feature of Arcana, though it doesn't seem to be in there at the moment. I added an issue to Arcana to implement that feature: https://github.com/microsoft/arcana.cpp/issues/4. Can you add a comment or issue (or both) here to remind us to switch to using that feature once it's available?